### PR TITLE
[Docs] Fix syntax

### PR DESCRIPTION
**Pull request by chenglou**

### CODE REVIEW

**Comment by chenglou** [comment]
cc @cullophid

File: README.md [hunk 1]
```diff
@@ -33,50 +32,50 @@ let styles = Css.{
...
 
-```
+```reason
 let bounce = Css.keyframes([
-  ("0%": [ transform( scale(0.1, 0.1) ),  opacity(0.0) ]),
-  ("60%": [ transform( scale(1.2, 1.2) ),  opacity(1.0) ]),
-  ("100%": [ transform( scale(1.0,1.0) ), opacity(1.0) ])
+  {"0%": [ transform( scale(0.1, 0.1) ),  opacity(0.0) ]},
+  {"60%": [ transform( scale(1.2, 1.2) ),  opacity(1.0) ]},
+  {"100%": [ transform( scale(1.0,1.0) ), opacity(1.0) ]}
```
Comment (cullophid) [inline README.md]: The bug was actually the `:`... its supposed to be a tuple :)

**Comment by cullophid** [comment]
@chenglou Thanks! there was one small issue above

**Comment by chenglou** [comment]
Done!

**Comment by cullophid** [comment]
@chenglou  Thanks!

**Pull request merged**